wiki/Beatmap/Beatmapset/en.md

```diff
@@ -0,0 +1,15 @@
+---
+stub: true
+---
+
+# Beatmapset
```

Comment:
this is wrong. this is a term which should not exist.

Reply:
a beatmap contains difficulties, a "beatmapset" is a legacy internal term we use to collect difficulties of a beatmap. it has no place in casual discussion. use the terms "beatmap" and "difficulties" like we do everywhere on osu-web.

Reply:
if it shouldn't exist, why don't we remove the article as a whole and make a redirect to "Beatmap" instead with a small note about its historical/internal value? also, what about beatmapID/beatmapsetID articles, as we have these, too?

as for the term usage, I'll address it in a while

Reply:
These IDs should also be internal. They are not intended to be used or consumed by the user. They should be documented on the API wiki or in the code.